Gordon360/Services/AccountService.cs

```diff
@@ -9,6 +9,8 @@
 using System.Threading.Tasks;
 using Gordon360.Extensions.System;
 using Gordon360.Enums;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
```

Comment:
```suggestion
using Microsoft.EntityFrameworkCore.Metadata.Internal;
```
Do we need this new import?